kingpin/actors/rightscale/test/test_base.py

```diff
@@ -130,22 +130,16 @@ def test_generate_rightscale_params(self):
 
     def test_generate_rightscale_params_with_array(self):
         self.maxDiff = None
-        params = {'name': 'unittest-name',
-                  'status': 'enabled',
-                  'elasticity_params': {
-                      'schedule': [
-                          {'day': 'Sunday', 'max_count': 2,
-                           'min_count': 1, 'time': '07:00'},
-                          {'day': 'Monday', 'max_count': 2,
-                           'min_count': 1, 'time': '07:00'},
-                          {'day': 'Tuesday', 'max_count': 2,
-                           'min_count': 1, 'time': '07:00'}
-                      ]
-                  }}
-        expected_params = [
-            ('server_array[status]', 'enabled'),
-            ('server_array[name]', 'unittest-name'),
+        schedule = [
+            {'day': 'Sunday', 'max_count': 2, 'min_count': 1, 'time': '07:00'},
+            {'day': 'Monday', 'max_count': 2, 'min_count': 1, 'time': '07:00'},
+            {'day': 'Tuesday', 'max_count': 2, 'min_count': 1, 'time': '07:00'}
```

Comment:
What is inside of this python code doesn't really matter because in reality the code is parsed from the JSON file, and cannot be specified as an OrderedDict. As far as I'm aware RightScale schedule is the only one that requires particular order of parameters.